### DISCUSSION TITLE
Context action to add # pyright: ignore [WhatEverTypeIssueIsTheProblem]

### DISCUSSION THREAD

**Discussion by abenko-exscientia** [InSyncWithFoo/pyright-for-pycharm#31] [Ideas]
Hi, small feature-request/proposal, it'd be really handy to add a context action that would just add `# pyright: ignore [WhatEverTypeIssueIsTheProblem]` at the end of the line 🙂. Great plugin!

**Comment by InSyncWithFoo** [comment]
As it happens, I'm also working on this very feature. Converting to an issue (#32) so that the progress can be tracked.

**Reply by InSyncWithFoo** [reply]
Implemented in [`2c9b87f`](https://github.com/InSyncWithFoo/pyright-for-pycharm/commit/2c9b87fe14577cd2c91d10a76269a8bea3fb3a78). A demo can be downloaded from [here](https://github.com/InSyncWithFoo/pyright-for-pycharm/actions/runs/9040021362).

**Reply by abenko-exscientia** [reply]
Great work! Thank you.

**Reply by InSyncWithFoo** [reply]
(Just FTR, this is included in [0.3.0](https://github.com/InSyncWithFoo/pyright-for-pycharm/releases/tag/v0.3.0), which I just published. It should be approved and appear on the Marketplace in less than 2 days.)

**Reply by abenko-exscientia** [reply]
I already installed and tested it, works like a charm 👍
